core/object.cpp

```diff
@@ -1477,7 +1477,13 @@ Error Object::connect(const StringName &p_signal, Object *p_to_object, const Str
 			s->slot_map[target].reference_count++;
 			return OK;
 		} else {
-			ERR_FAIL_V_MSG(ERR_INVALID_PARAMETER, "Signal '" + p_signal + "' is already connected to given method '" + p_to_method + "' in that object.");
+			String msg = "Signal '" + p_signal + "'";
+			msg += " from \"" + _get_name_internal() + "\" ";
+			msg += "is already connected to given method '" + p_to_method + "' in ";
+			msg += "\"" + p_to_object->_get_name_internal() + "\"";
+			msg += ".";
+
+			ERR_FAIL_V_MSG(ERR_INVALID_PARAMETER, msg);
```

Comment:
This would be much better using `vformat` IMO.
```suggestion
			ERR_FAIL_V_MSG(ERR_INVALID_PARAMETER,
					vformat("Signal \"%s\" from \"%s\" is already connected to given method \"%s\" in \"%s\".",
							p_signal, _get_name_internal(), p_to_method, p_to_object->_get_name_internal()));
```

Reply:
But that change + the added virtual method in `core/object.h` should likely go to their own PR, they're outside the scope of this change and likely the kind of thing that requires a lot more bikeshedding.

Reply:
Ah yes, I added this because of a bug I was investigating at the time. It is pretty handy because the existing message doesn't tell you where the signal is coming from if I remember right. But I'll move to a later PR. :+1: